src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2.java

```diff
@@ -90,6 +91,7 @@
         oneLineSummary = "Call somatic SNPs and indels via local re-assembly of haplotypes",
         programGroup = VariantProgramGroup.class
 )
+@DocumentedFeature
```

Comment:
No, I'd rather have all tools show up unless explicitly tagged to be ignored. But conversely it would be good to be able to tag tools as being experimental or in development, eg specifying alpha or beta status. 

Actually, do/can we have a label system that allows us to use arbitrary tags? Something that we could use web-side to would allow users to filter tools by tag, eg "only show me alpha tools" or "only show me production-worthy tools", or "show me coverage tools"... Would like like eg `programTags = [alpha, coverage]` where alpha and coverage are defined in some curated class somewhere (so it doesn't become a wild wild west of alternative phrasings or spellings).